scripts/start.mjs

```diff
@@ -0,0 +1,35 @@
+import { spawn, exec } from "child_process";
+console.log("Starting a hardhat node...");
+const hardhat = spawn("yarn contracts hardhat node", { shell: true });
+
+hardhat.stdout.on("data", (data) => {
+    // when start an Account list
+    if (data.toString().indexOf("========") !== -1) {
+        // deploy a Unirep contract and the attester contract
+        exec("yarn contracts deploy", (err) => {
+            if (err) {
+                console.error(`Deploy contract error: ${err}`);
+                process.exit(1);
+            }
+            console.log("Contract has been deployed");
+
+            // start a relayer
+            console.log("Start a relayer");
+            const relay = spawn("yarn relay start", { shell: true });
```

Comment:
We need to handle the relay/frontend exiting. I think for now if either process exits we should exit the top level process. The problem is the blockchain state will be lost. So it might be nice to give the user the option of restarting the relay/frontend if they exit (without restarting the whole process).